src/Microsoft.Health.Fhir.SqlServer.Api/Features/Filters/NullArgumentExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.SqlServer.Api.Features.Filters
+{
+    public class NullArgumentExceptionFilterAttribute : ActionFilterAttribute
```

Comment:
This seems like an overly broad approach to capture attempting getting a script that doesn't exist. This will likely capture anywhere in the stack that an `ArgumentNullException` occurs. We should probably update the code where it's failing to do a null check and throw a specific error type to handle. 

We can then implement a more general filter similar to `OperationOutcomeExceptionFilterAttribute` for just the schema controller. That would allow us to combine this logic with the `NotImplementedExceptionFilter` and handle the output in a standard manner.

Reply:
addressed it 👍